test/test_rack_session.rb

```diff
@@ -252,10 +252,10 @@
 
     res0 = req.get("/")
     session_id = (cookie = res0["Set-Cookie"])[session_match, 1]
-    ses0 = rsd.pool.get(session_id, true)
+    ses0 = rsd.pool.get(session_id, cache_nils: true)
```

Comment:
@yangxing-star How are the changes in test_rack_session.rb relevant to the PR?  I don't understand why this was changed from `true` to `{ cache_nils: true }`